Resources/Prototypes/Catalog/VendingMachines/Inventories/cargodrobe.yml

```diff
@@ -4,16 +4,21 @@
     ClothingBackpackCargo: 3
     ClothingBackpackSatchelCargo: 3
     ClothingBackpackDuffelCargo: 3
-    ClothingUniformJumpsuitCargo: 3
-    ClothingUniformJumpskirtCargo: 3
+    ClothingUniformJumpsuitCargoLight: 3 # CD
+    ClothingUniformJumpsuitCargoDark: 3 # CD
+    ClothingUniformJumpskirtCargoLight: 3 # CD
+    ClothingUniformJumpskirtCargoDark: 3 # CD
     ClothingUniformJumpsuitCargoPants: 3 # CD
     ClothingShoesColorBlack: 3
     ClothingHandsGlovesFingerless: 3
+    ClothingHandsGlovesMechanic: 3 # CD
     ClothingHeadHatCargosoft: 3
     ClothingHeadBandBrown: 3
+    ClothingMaskBandCargo : 3 # CD
     ClothingHeadsetCargo: 3
     ClothingOuterWinterCargo: 2
     ClothingOuterWinterMiner: 2
     ClothingOuterCoatCargoBomber: 2 # CD
     ClothingNeckScarfStripedBrown: 3
     ClothingShoesBootsWinterCargo: 2
+    ClothingShoesBootsSteel: 2 # CD
```

Comment:
Groups all of the CD items together to reduce merge conflicts.

Reply:
It's like this to preserve the order they're shown in the cargodrobe though, not sure what you mean

Reply:
Oh, then that is fine. By merge conflicts I am referring to the conflicts created when Lank merges upstream. Having all of the CD changes in one block helps with that.